src/Compiler/pars.fsy

```diff
@@ -4788,6 +4789,14 @@ argExpr:
         arg }
 
 atomicExpr:
+  | UNDERSCORE DOT atomicExpr %prec dot_lambda
+      { let arg1 = rhs parseState 1
```

Comment:
Nit, but `mUnderscore` and `mDot` would be better than `arg1` and `arg2`.
`arg3` could be `expr`.

Reply:
Good point, will change